### PR TITLE
Chicago taxi fare prediction

### PR DESCRIPTION
**Pull request by sudarshan-SpringML**
If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [ ] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [ ] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under `# Official Notebooks` section, pointing to the author or the author's team.
- [ ] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under the `# Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).

If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under the `# Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).

### CODE REVIEW

File: notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb [hunk 1]
```diff
@@ -0,0 +1,1440 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb:1]: <p>Table of contents and Objective sectios</p><ul><li>Title and Bullets need to be in sentence format (only first word capitialized).</li></ul><p> </p><ul><li>Vertex-AI -&gt; Vertex AI</li><li>Explanations -&gt; Vertex Explainable AI</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (andrewferlitsch) [inline notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb:1]: <p>should follow notebook template for setting PROJECT</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb:1]: <p>if BUCKET_NAME == "" or BUCKET_NAME == "[your-bucket-name]" or BUCKET_NAME is None:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (andrewferlitsch) [inline notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb:1]: <p>Vertex-AI -&gt; Vertex AI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (andrewferlitsch) [inline notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb:1]: <p>charts(distribution plots) : Missing a space</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (andrewferlitsch) [inline notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb:1]: <p>as it is -&gt; as-is</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Comment (andrewferlitsch) [inline notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb:1]: <p>Vertex-AI -&gt; Vertex AI</p><p>cloud storage -&gt; Cloud Storage</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Comment (andrewferlitsch) [inline notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb:1]: <p>Vertex-AI -&gt; Vertex AI</p><p>Explanations -&gt; Vertex Explainable AI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Comment (andrewferlitsch) [inline notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb:1]: <p>BUCKET_NAME -&gt; BUCKET_URI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='93'/>

Comment (sudarshan-SpringML) [reply notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb:1]: <p>resolved</p>

Comment (sudarshan-SpringML) [reply notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb:1]: <p>resolved</p>

Comment (sudarshan-SpringML) [reply notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb:1]: <p>resolved</p>

Comment (sudarshan-SpringML) [reply notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb:1]: <p>resolved</p>

Comment (sudarshan-SpringML) [reply notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb:1]: <p>resolved</p>

Comment (sudarshan-SpringML) [reply notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb:1]: <p>resolved</p>

Comment (sudarshan-SpringML) [reply notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb:1]: <p>resolved</p>

Comment (sudarshan-SpringML) [reply notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb:1]: <p>resolved</p>

Comment (sudarshan-SpringML) [reply notebooks/community/managed_notebooks/chicago_taxi_fare_prediction/chicago_taxi_fare_prediction_notebook.ipynb:1]: <p>resolved</p>

**Pull request merged**